test/specs/equipment.ts

```diff
@@ -0,0 +1,67 @@
+import { expect } from "@wdio/globals";
+import MainPage from "../pageobjects/main.page.ts";
+import UnitPage from "../pageobjects/unit.page.ts";
+import ProductsPage from "../pageobjects/products.page.ts";
+import * as testData from "../data/testdata.json";
+
+describe("Rentzila", () => {
+  it('C213- Checking ""Спецтехніка"" section on the main page', async () => {
+    await MainPage.open();
+
+    await expect(MainPage.equipmentTitle).toBeExisting();
+    await expect(MainPage.equipmentTitle).toHaveText("Спецтехніка");
+
+    await verifyEquipment(
+      MainPage.popularEquipmentLabel,
+      testData.equipment.popular.names,
+      testData.equipment.popular.namesInFilters,
+      testData.equipment.popular.namesInUnitPage
+    );
+    await verifyEquipment(
+      MainPage.agriculturalEquipmentLabel,
+      testData.equipment.agricultural.names,
+      testData.equipment.agricultural.namesInFilters,
+      testData.equipment.agricultural.namesInUnitPage
+    );
+    await verifyEquipment(
+      MainPage.buildingEquipmentLabel,
+      testData.equipment.building.names,
+      testData.equipment.building.namesInFilters,
+      testData.equipment.building.namesInUnitPage
+    );
+    await verifyEquipment(
+      MainPage.otherEquipmentLabel,
+      testData.equipment.other.names,
+      testData.equipment.other.namesInFilters,
+      testData.equipment.other.namesInUnitPage
+    );
+  });
+  async function verifyEquipment(
```

Comment:
@Anastas11aZakharova That is a greate decision, to use fulction in the test spec like that< to reduce the amount of code! But, fore such a HUUUGE functions, please, add a JSDoc/TSDoc, even if they are directly in the spec files, like that, since when someone will check the code and its correspondance to the documentation, or when someone will fix that after some updates to the UI - the description for the function/method will reduce some time for the review/maintain processes.

Reply:
Fixed in the latest commit